meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/util/ZkMetaClientUtil.java

```diff
@@ -210,4 +222,87 @@ public static MetaClientException translateZkExceptionToMetaclientException(ZkEx
       return new MetaClientException(e);
     }
   }
+
+  /**
+   * This function translate and group Zk exception code to metaclient code.
+   * It currently includes all ZK code on 3.6.3.
+   */
+  public static MetaClientException.Code translateKeeperCodeToMetaClientCode(
```

Comment:
NIT: that's too much as keepercode. Most of zookeeper user even dont know what is keepercode. Better we call it fromZookeeperCodeToMetaClientCode()